sdk/core/abort-controller/README.md

```diff
@@ -8,9 +8,9 @@ in an Azure SDK that accept a parameter of type `AbortSignalLike`.
 
 Key links:
 
- - [Source code](https://github.com/Azure/azure-sdk-for-js/edit/master/sdk/core/abort-controller)
- - [Package (npm)](https://www.npmjs.com/package/@azure/abort-controller)
- - [API Reference Documentation](https://docs.microsoft.com/javascript/api/overview/azure/abort-controller-readme)
+- [Source code](https://github.com/Azure/azure-sdk-for-js/tree/main/sdk/core/abort-controller)
```

Comment:
huh looks like /edit/main/sdk/core/abort-controller will redirect correctly to /tree/main/sdk/abort-controller but not for master. Maybe it got recently deleted? 

Anyway this is the correct link to use